spec/smile-identity-core/id_api_spec.rb

```diff
@@ -1,16 +1,16 @@
 # frozen_string_literal: true
 
 RSpec.describe SmileIdentityCore::IDApi do
-  let(:partner_id) { '001' }
-  let(:api_key) { Base64.encode64(OpenSSL::PKey::RSA.new(1024).public_key.to_pem) }
-  let(:sid_server) { 0 }
-  let(:connection) { described_class.new(partner_id, api_key, sid_server) }
+  let(:partner_id) { ENV.fetch('SMILE_PARTNER_ID') }
+  let(:api_key) { ENV.fetch('SMILE_API_KEY') }
+  let(:sid_server) { ENV.fetch('SMILE_SERVER_ENVIRONMENT', 0) }
+  let(:connection) { SmileIdentityCore::IDApi.new(partner_id, api_key, sid_server) }
 
   let(:partner_params) do
     {
-      user_id: 'dmKaJazQCziLc6Tw9lwcgzLo',
-      job_id: 'DeXyJOGtaACFFfbZ2kxjuICE',
-      job_type: SmileIdentityCore::JOB_TYPE::BASIC_KYC
+      user_id: SecureRandom.uuid,
+      job_id: SecureRandom.uuid,
+      job_type: SmileIdentityCore::JOB_TYPE::ENHANCED_KYC
```

Comment:
this is because `BASIC_KYC` returns "Match / No Match / Partial Match"

correct?

Reply:
That is right